GameData/ROEngines/PartConfigs/AJ60_BDB.cfg

```diff
@@ -0,0 +1,93 @@
+PART
+{
+	module = Part
+	name = ROE-AJ60
+	author = Cobaltwolf, Vader111
+	
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+	
+	mass = 3.9384
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+	
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/AtlasV/bluedog_AtlasV_AJ60
+	}
+	
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_attach = 0, 0.5, 0.491, 0.0, 0.0, -1.0, 1
+	CoMOffset = 0.0, -0.245, 0.245
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0	
+	
+	title = AJ-60A
+	manufacturer = Aerojet Rocketdyne
+	description = AJ-60A is a solid rocket booster produced by Aerojet Rocketdyne. They are currently used as strap-on boosters on United Launch Alliance's Atlas V rocket.
+	
+	tags = aerojet, AJ60, booster, radial, solid, srb,
+	
+	engineType = AJ60A
+	stagingIcon = SOLID_BOOSTER
+	stageOffset = 1
+	childStageOffset = 1
+	
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 200
+	breakingTorque = 200
+	crashTolerance = 7
+	
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
```

Comment:
Make use of ROESetEngineDefaults  (and beware those duplicate definitions of breakingForce and breakingTorque from above!)